readme-developer.txt

```diff
@@ -85,12 +85,30 @@ To build Endless Sky with native tools, you will first need to download Xcode fr
 
 Next, install Homebrew (from http://brew.sh).
 
+ARM Processors:
+
+  If you have an ARM processor, such as the Apple M1, you will need to install an x86_64 version of Homebrew. You can do this by running the normal install command through an `arch` terminal:
+
+  $ arch -x86_64 zsh
+
+  You may find it useful to define an alias that you can use to interact with your x86_64 homebrew:
+
+  alias brex="arch -x86_64 /usr/local/Homebrew/bin/brew"
```

Comment:
A readme like this should describe the minimal amount of steps to compile for a platform. Power-user optimizations like defining an alias are not needed for power-users that already know about aliases, and non-power-users could easily be confused by an alias definition.